### PR TITLE
OSDOCS-4148: Release note for "Support kubernetes-nmstate on vSphere platforms"

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
A release note for 4.12, Networking, supporting k8s-nmstate on VMWare vSphere

Issue: [OSDOCS-4148](https://issues.redhat.com//browse/OSDOCS-4148)

Version(s):
  * PR applies to: 4.12 only

Link to docs preview:
https://51249--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-networking

Doc change:
https://github.com/openshift/openshift-docs/pull/51746

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@creydr and @qiowang721 Please review release note and either approve or suggest revisions. Thank you!

**Comment by creydr** [comment]
Works for me. But we need to wait on the network manager fix and then the confirmation from QE

**Comment by ShaunaDiaz** [comment]
@creydr Great, please add lgtm

**Comment by qiowang721** [comment]
works for me.
/lgtm

**Comment by creydr** [comment]
@ShaunaDiaz will you update https://docs.openshift.com/container-platform/4.11/networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.html too, that vSphere is supported?

**Comment by ShaunaDiaz** [comment]
@creydr Yes, Travis building it now. When it's ready, I will @ you.

**Review by creydr — APPROVED** [review]
/lgtm

**Review by tmalove — COMMENTED** [review]
/remove-label peer-review-needed
/label peer-review-in-progress

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -81,6 +81,12 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-networking"]
 === Networking
 
+[id="ocp-4-12-k8s-nmstate-support-for-vsphere"]
+==== Kubernetes NMState in VMWare vSphere now supported
+Beginning with OpenShift Container Platform 4.12, you can configure the networking settings such as DNS servers/search domains, VLANs, bridges, and interface bonding using the Kubernetes NMState Operator on your VMware vSphere instance.
```
Comment (tmalove) [inline release_notes/ocp-4-12-release-notes.adoc:86]:
```suggestion
Beginning with {product-title} 4.12, you can configure the networking settings such as DNS servers or search domains, VLANs, bridges, and interface bonding using the Kubernetes NMState Operator on your VMware vSphere instance.
```
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-slashes

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -81,6 +81,12 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
...
+
+For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc
```
Comment (tmalove) [inline release_notes/ocp-4-12-release-notes.adoc:88]: This link needs to be active.

**Review by tmalove — COMMENTED** [review]
/remove-label peer-review-in-progress
/label peer-review-done

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (ShaunaDiaz) [reply release_notes/ocp-4-12-release-notes.adoc]: Fixed these, thank you1

File: release_notes/ocp-4-12-release-notes.adoc (hunk 2 above)
Comment (ShaunaDiaz) [reply release_notes/ocp-4-12-release-notes.adoc]: Fixed, thank you!

**Review by creydr — COMMENTED** [review]
/lgtm

**Pull request merged**